### ISSUE TITLE
Remote Models with dashes in the name cause an exception.

### ISSUE THREAD

**Issue by ProjectMoon** [Jeffser/Alpaca#159]
**Describe the bug**
If a model has a dash in the name (not tag), calls to `/api/chat` fail for some reason. So a model like `my-model:latest` will fail, but not `mymodel:lat-test`.

I am not sure if this happens on localhost ollama. But it happens to me on both direct http connections over LAN and HTTPS connections over WAN.

**Expected behavior**
Model responds.

**Screenshots**
This happens after attempting to talk to a model like `mistral-nemo:instruct`. But a model named `mistral:nemo` works just fine.

![image](https://github.com/user-attachments/assets/90aa7e50-126c-4089-9b81-52533683b51f)

**Additional context**
At first, I thought this was OpenWebUI having issues, but it also happens when connecting directly to the ollama server remotely. I proxy ollama via OpenWebUI for external connections. The same behavior occurs when using either OpenWebUI's ollama proxy API, or talking to ollama directly. Notably, there is a pretty big mess of exceptions in the OpenWebUI log when calling a model with a dash in it.

**Comment by Jeffser** [comment]
I'm looking into this, thanks for the report

**Comment by Jeffser** [comment]
yeah I found out what is causing this error, normally when a model has a '-' it gets replaced with a space, but it doesn't revert that space back to '-' when sending a request to Ollama, so it doesn't know what to do with a model with a space in the name.

The fix should be ready and pushed to Flathub in a couple of hours!

**Comment by Jeffser** [comment]
[Fixed!](https://github.com/Jeffser/Alpaca/commit/a56631510d4f35c716bb1b8ca925e9db13edccb5)

![image](https://github.com/user-attachments/assets/01471646-91c7-4869-b4f8-9d99806a45f9)
